crates/holochain/src/conductor/interface.rs

```diff
@@ -53,13 +53,17 @@ pub struct SignalBroadcaster {
 }
 
 impl SignalBroadcaster {
-    /// send the signal to the connected client
+    /// Send the signal to the connected client. Send error is logged but not
+    /// returned to the caller.
     pub(crate) fn send(&mut self, sig: Signal) -> InterfaceResult<()> {
-        self.senders
-            .iter_mut()
-            .map(|tx| tx.send(sig.clone()))
-            .collect::<Result<Vec<_>, broadcast::error::SendError<Signal>>>()
-            .map_err(InterfaceError::SignalSend)?;
+        self.senders.iter_mut().for_each(|tx| {
+            if let Err(err) = tx.send(sig.clone()) {
+                tracing::error!(
```

Comment:
I wouldn't say an expected part of operation, rather a mistake that hasn't been noticed before. But I'm fine with demoting this to info or debug. Any other opinions?

Reply:
I'm also wondering about the 'no active receivers connected' component of the log message. even I wouldn't know what to make of that.

Reply:
Yeah, it felt really correct at the time of writing because it's what the inner tokio error says, but no one will understand it. I've changed that to "no app websocket connected". I want to output the port number as well, but that requires some refactoring which I may be able to do next week.